.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Thanks for your comments! I see your concern about robustness and they are certainly valid to keep the codebase clean. If a test is renamed and should've been skipped, then the test won't be skipped unless the list is updated and this would hopefully all the developer to check for that.

As for a test being deleted entirely, the skip interface won't break any tests but there's no easy way to know that a test wasn't present. I could in theory as part of the interface make it print out any tests that were listed to be skipped but not actually skipped before the test suite is run. This would allow us to know if there are stray tests in the array no longer needed. Thoughts? -- I suppose this would help with the renamed tests as well.